README.md

```diff
@@ -443,6 +444,15 @@ Metrics/CyclomaticComplexity:
   Severity: warning
 ```
 
+## Details 
+
+Individual cops can be embellished with extra details in offense messages:
+
+```yaml
+Metrics/LineLength:
+  Details: You can thank the IBM punch card for the 80 character limit :-)
```

Comment:
I think you could choose a different message for this example. One that can't be viewed as arguing against RuboCop's defaults. Am I being too sensitive?